File: 3498-reverse-degree-of-a-string/3498-reverse-degree-of-a-string.py
class Solution:
    def reverseDegree(self, s: str) -> int:
        l=[]
        for i in range(26):
            l.append(26-i)
        sum=0
        for i in range(len(s)):
            sum+=(i+1)*(l[ord(s[i])-ord('a')])
        return sum

File: 3498-reverse-degree-of-a-string/3498-reverse-degree-of-a-string.java
class Solution {
    public int reverseDegree(String s) {
        int count=0;
        for(int i=0; i<s.length(); i++){
            int val=Math.abs('z'-s.charAt(i)+1);
            count+=(val*(i+1));
        }
        return count;
    }
}